clients/vscode-hlasmplugin/scripts/syntaxes/hlasm_tmgrammar_template.txt

```diff
@@ -400,8 +400,12 @@
       }
     },
     "remark": {
-      "match": "(\\S+)(?<=^${listingDetails}$.{0,71})",
-      "name": "comment.hlasm"
+      "match": "\\s*(\\S.*)(?<=^${listingDetails}$.{0,71})",
```

Comment:
maybe
```suggestion
      "match": "(\\S.*)(?<=^${listingDetails}$.{0,71})",
```
?

Reply:
I needed to also include `${listingDetails}$` but yeah 👍